app/assets/stylesheets/layout.scss

```diff
@@ -1,3 +1,10 @@
+@font-face {
+  font-family: 'Inter';
+  src: url(asset-path('Inter.ttf')) format('truetype');
```

Comment:
```suggestion
  src: font-url('Inter.ttf') format('truetype');
```

Reply:
Shouldn't it be `fonts/Inter.tff` instead of just `Inter.tff` ?

Reply:
using asset-path() inside url() or directly using font-url() both works fine 
we have to keep path only as 'Inter.ttf'... if we  use fonts/Inter.ttf we get path such as ' localhost:3000/fonts/fonts/Inter.ttf'
and we end with 404 error as font-url() will take care of fonts folder we just need 'Inter.ttf' as path...